redact-wasm/README.md

```diff
@@ -0,0 +1,78 @@
+<a href="https://arcjet.com" target="_arcjet-home">
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="https://arcjet.com/logo/arcjet-dark-lockup-voyage-horizontal.svg">
+    <img src="https://arcjet.com/logo/arcjet-light-lockup-voyage-horizontal.svg" alt="Arcjet Logo" height="128" width="auto">
+  </picture>
+</a>
+
+# `@arcjet/analyze`
+
+<p>
+  <a href="https://www.npmjs.com/package/@arcjet/analyze">
+    <picture>
+      <source media="(prefers-color-scheme: dark)" srcset="https://img.shields.io/npm/v/%40arcjet%2Fanalyze?style=flat-square&label=%E2%9C%A6Aj&labelColor=000000&color=5C5866">
+      <img alt="npm badge" src="https://img.shields.io/npm/v/%40arcjet%2Fanalyze?style=flat-square&label=%E2%9C%A6Aj&labelColor=ECE6F0&color=ECE6F0">
+    </picture>
+  </a>
+</p>
+
+[Arcjet][arcjet] helps developers protect their apps in just a few lines of
+code. Implement rate limiting, bot protection, email verification, and defense
+against common attacks.
+
+This is the [Arcjet][arcjet] Redaction library PII detection engine.
+
+## Installation
+
+```shell
+npm install -S @arcjet/redact-wasm
+```
+
+## Example
+
+```ts
+import { generateFingerprint, isValidEmail } from "@arcjet/analyze";
+
+const fingerprint = generateFingerprint("127.0.0.1");
+console.log("fingerprint: ", fingerprint);
+
+const valid = isValidEmail("hello@example.com");
+console.log("is email valid?", valid);
+```
+
+## Implementation
+
+This package provides pii identification logic implemented as a WebAssembly module which
+will run local analysis on the provided string.
+
+The [arcjet.wasm.js](./wasm/arcjet_analyze_bindings_redact.component.js) file contains the binary inlined as
```

Comment:
```suggestion
The [Wasm](./wasm/arcjet_analyze_bindings_redact.component.js) file contains the binary inlined as
```

Reply:
I actually realized that this section is totally out of date because we use rollup to output in the _virtual directory. Let me think through rewording it.